bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/GroovyScriptEngineFactory.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
```

Comment:
Please use the license header from https://github.com/openhab/openhab-core/blob/master/licenses/epl-2.0/header.txt

Reply:
I'm not sure if I should use this...

    Copyright (c) 2010-${year} Contributors to the openHAB project

Or...

    Copyright (c) 2010-2019 Contributors to the openHAB project

Reply:
Clearly the version with the years, not the placeholders!

Reply:
I thought so... but clearly not so clear to me! Thank you! :smile: